test/test.js

```diff
@@ -44,6 +42,42 @@ describe('CheckedSelect behaviour independent of synchronicity', () => {
                 // then
                 assert.isTrue(changeHandler.notCalled);
             });
+
+            it('displays the placeholder text once if nothing is selected', () => {
+                // given
+                const options = getMinimalOptionsProp();
+                const placeholderText = 'Some number';
+                //when
+                const wrapper = render(<CheckedSelect
+                    async={ asyncValue }
+                    options={ makeOpts(options) }
+                    loadOptions={ makeLoadOpts(options) }
+                    onChange={ sinon.stub() }
+                    value={ [] }
+                    placeholder={ placeholderText }
+                />);
+                // then
+                const placeholderRegExp = new RegExp(placeholderText, 'g');
+                assert.lengthOf(wrapper.text().match(placeholderRegExp), 1);
```

Comment:
How are we sure that the rendered component contains only the placeholderText? Should we do the exact matching instead? Because if you only match it with the content, isn't it there will be possibility the placeholder will contain other text?

Reply:
I'm not sure if it would be feasible to test that on this level, as an input element like this one could potentially contain additional text that's styled to only appear to assistive technologies such as screen readers.

Reply:
Although it does seem like, for now, Enzyme does return exactly what we'd expect. I'll simplify the test to an `assert.equal`; if it starts failing for whatever reason we can make it more subtle later.